src/utils/maps/loadGoogleMapsLibrary.ts

```diff
@@ -0,0 +1,24 @@
+const API_URL = 'https://maps.googleapis.com/maps/api/js';
```

Comment:
since this file is only used on the address search input let's move it to the folder there. Unless we start having shared hooks let's avoid spreading the logic.